themes/mainroad/layouts/partials/colors.css

```diff
@@ -56,8 +56,13 @@ figcaption {
 
 pre,
 code {
-	background-color: var(--color-box);
-	border: 1px solid var(--color-border);
+	background-color: var(--color-code-background);
+	border-radius: 0.2em;
```

Comment:
actually, hol up - wouldn't it be smarter to use `rem` instead? rem was also used in the other css file, so I'm not sure why it wasn't used here

Reply:
Because code tag is inline and its size can change based on parent tag. em depends on current tag font size, rem depends on root tag font size. so it would make sense to scale it if you have H1 code tag for example. It doesnt usually happen, but /shrug